From 95a87bf075155e70828a19508722fb74f8e28bf5 Mon Sep 17 00:00:00 2001
Message-Id: <95a87bf075155e70828a19508722fb74f8e28bf5@dist-git>
From: =?UTF-8?q?J=C3=A1n=20Tomko?= <jtomko@redhat.com>
Date: Mon, 7 Apr 2014 13:30:52 +0200
Subject: [PATCH] qemu: fix default spice password setting

6.6: https://bugzilla.redhat.com/show_bug.cgi?id=953721

Set spice password even if default VNC password hasn't been set.

https://bugzilla.redhat.com/show_bug.cgi?id=953720
(cherry picked from commit 4327df7eee75c5bec485703eb5b5f439f2fd371f)

Conflicts:
	src/qemu/qemu_hotplug.c - missing commit b090aa7:
        Introduce a virQEMUDriverConfigPtr object

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/qemu/qemu_hotplug.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qemu/qemu_hotplug.c b/src/qemu/qemu_hotplug.c
index ab38a39..5849e13 100644
--- a/src/qemu/qemu_hotplug.c
+++ b/src/qemu/qemu_hotplug.c
@@ -2626,7 +2626,7 @@ qemuDomainChangeGraphicsPasswords(struct qemud_driver *driver,
     time_t lifetime;
     int ret;
 
-    if (!auth->passwd && !driver->vncPassword)
+    if (!auth->passwd && !defaultPasswd)
         return 0;
 
     if (auth->connected)
-- 
1.9.1

